static/js/notifications.js

```diff
@@ -164,24 +163,23 @@ export function is_window_focused() {
 }
 
 export function notify_above_composebox(
-    note,
-    link_class,
+    banner_text,
+    classname,
+    link_text,
     above_composebox_narrow_url,
     link_msg_id,
-    link_text,
 ) {
     const $notification = $(
         render_compose_notification({
-            note,
-            link_class,
+            banner_text,
+            classname,
+            link_text,
             above_composebox_narrow_url,
             link_msg_id,
-            link_text,
         }),
     );
     clear_compose_notifications();
-    $("#out-of-view-notification").append($notification);
-    $("#out-of-view-notification").show();
```

Comment:
This no longer shows and hides -- the container is always visible, similar to the other new compose banners.